prow/pjutil/pprof.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package pjutil contains helpers for working with ProwJobs.
+package pjutil
+
+import (
+	"net/http"
+	"net/http/pprof"
```

Comment:
I think this will also have the side effect of registering handlers to the default mux for any process that imports `pjutil`. This is probably not a problem, but something to keep in mind. I don't think there is a way to avoid this anyways.

Reply:
Yeah, I did not see any way since it's an `init()`